README.md

```diff
@@ -443,3 +443,32 @@ wit`. You can link two components together with `wasm-tools compose` as well.
 Note that the runtimes above are generally intended to work with arbitrary
 components, not necessarily only those created by `wit-bindgen`. This is also
 not necessarily an exhaustive listing of what can execute a component.
+
+## Building and Testing
+
+To build the cli:
+
+```
+cargo build
+```
+
+There are a few perquisites to testing the project. You only need the language compilers that you wish to run test against.
+
+- WASI SDK
+  - Download from wasi-sdk releases page. If you're using Windows, you need the one with mingw in its name.
+  - `curl -LO https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-20/wasi-sdk-20.0-linux.tar.gz`
+  - Create an environment variable called `WASI_SDK_PATH`` giving the path where you extracted the WASI SDK download, i.e., the directory containing `bin`/`lib`/`share`` folders.
+- Compilers for the target language:
+  - TinyGo - https://tinygo.org/
```

Comment:
Can you add that both Go and TinyGo are needed? I don't remember the minimal version for TinyGo, but have been specified in Readme somewhere.

Reply:
I missed the go dep since I had it installed already. Thanks!